demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -17,6 +17,7 @@
 import cv2
 import numpy as np
 from argparse import ArgumentParser, SUPPRESS
+from inpainting_gui import InpaintingGUI
```

Comment:
Please, reorder import according to [PEP](https://www.python.org/dev/peps/pep-0008/#imports) (standart libs, third party lib, local lib)

Reply:
Done